packages/js/product-editor/src/components/attribute-control/new-attribute-modal.tsx

```diff
@@ -58,6 +58,8 @@ type AttributeForm = {
 	attributes: Array< EnhancedProductAttribute | null >;
 };
 
+const MAX_N_TERMS_AUTO_SELECTION = 10;
```

Comment:
Just a small suggestion: I think that `MAX_TERMS_AUTO_SELECTION` would work here too. I know that it's slightly different so it's on you to keep it or change it.